group_vars/location_mlk_nk/dns.yml

```diff
@@ -0,0 +1,13 @@
+---
+
+dns_servers:
+  # quad9
+  - 9.9.9.9
+  - 149.112.112.112
+  - 2620:fe::fe
+  - 2620:fe::9
+  # cloudflare
+  - 1.1.1.1
+  - 1.0.0.1
+  - 2606:4700:4700::1111
+  - 2606:4700:4700::1001
```

Comment:
Many Freifunkers and I dislike cloudflare for beeing a US-Company and thus being in the sphere of Cloud-Act. We cannot be sure, that there will be no dataleakage, Internet-tracking or whatever.

While it may be okay to have cloudflare in ones Freifunka private network, I would really like to avoid using it in the backbone for the aforementioned reasons.